### PR TITLE
New Author: Esmerlin Moya

### PR DESCRIPTION
**Pull request by EsmerlinMoya**
:pencil2: **Blog Url**: https://medium.com/@esmerlinmoya

:scroll: **Feed Url**: https://medium.com/feed/@esmerlinmoya

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines (please check each item that applies):

- [x] I have a valid blog & RSS URL, both using HTTPS :lock: with a valid certificate
- [x] I host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- [x] My blog is active with at least 3 Xamarin related blog posts in the last 6 months
- [x] In addition to the previous guideline; it should be apperant that the blog is active for a longer period of time and the posts on it are not fabricated in a small amount of time to comply to the '3 posts in last 6 months' guideline.
- [ ] I have applied a filter (Optional: if blog is only about Xamarin or you use `xamarin` in the title or categories in your blog posts)
- [x] I have added my Author class to the csproj file
- [x] I have implemented `IAmACommunityMember` or one of the other derivatives
- [x] I understand, if I delete my blog, I will be deleted from Planet Xamarin
- [x] I understand, my blog may be removed at any time if any of these guidelines are broken.
- [x] The submitted blog is not owned by a company which sole purpose it is to advertise a product of productline.

Only the collaborators of this repository decide whether a blog is eligible or not. They will determine if the rules are followed and if the quality of the content is high enough. 

Opening another PR, after the first one is closed in a small period of time will not be reevaluated. If a PR is closed, the issues with your blog are too significant to be fixed within a matter of hours.

### CODE REVIEW

**Comment by EsmerlinMoya** [comment]
@jfversluis PR template and Its running correctly 

planscreen

File: src/Firehose.Web/Authors/EsmerlinMoya.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+	public class EsmerlinMoya : IAmACommunityMember
```
[the remark this answers is not in the document]
Comment (EsmerlinMoya) [reply src/Firehose.Web/Authors/EsmerlinMoya.cs]: @Cheesebaron Fixed, please Verify

[the remark this answers is not in the document]
Comment (EsmerlinMoya) [reply src/Firehose.Web/Authors/EsmerlinMoya.cs]: @Cheesebaron Clean up indentation Done, please verify

**Comment by EsmerlinMoya** [comment]
@jfversluis @Cheesebaron  since my Pr passed all check, can it be merge please?

**Comment by jfversluis** [comment]
Upon further inspection I see your blog has just started. You have checked the "My blog is active with at least 3 Xamarin related blog posts in the last 6 months" requirement, but you do not comply to it. The start looks good, but I'd like to see a bit more posts over a longer period of time. A lot of people start out with a blog and then never look at it again, that is why we have this rule in place.

Please keep up the good work, and try again in a few months when your blog is filled with awesome content.

**Comment by EsmerlinMoya** [comment]
@jfversluis Ok (Y)

**Pull request closed without merging**